### ISSUE TITLE
`comments-indentation` false positive

### ISSUE THREAD

**Issue by Vanav** [adrienverge/yamllint#574]
Error example:
```
- block:
  - name: Command 1
    command: test 1
  #- name: Command 2
  #  command: test 2
```
Output:
```
yaml[comments-indentation]: Comment not indented like content
roles/base/tasks/RedHat-test.yml:4
```
.yamllint:
```
rules:
  indentation:
    indent-sequences: consistent
  comments:
    require-starting-space: false
  comments-indentation: {}
```
No error:
```
- block:
  #- name: Command 2
  #  command: test 2
  - name: Command 1
    command: test 1
```
I'm using ansible-lint latest version 6.17.0.
Seems parser doesn't like comment block at the end of file..

**Comment by adrienverge** [comment]
Hello,

Could this be a duplicate of https://github.com/adrienverge/yamllint/issues/384 (and https://github.com/adrienverge/yamllint/issues/141)?

**Comment by Vanav** [comment]
Yes, I've not found it, I'm sorry.
